config/evm_contracts_deployment_v2/pharos-testnet.json

```diff
@@ -9563,6 +9563,1800 @@
+        "quote": null
+      },
+      "10129": {
+        "address": "0xFB05e9D90407Aae9d7E8659d803C42C0bE2C12e2",
+        "constructorArgs": [
+          "MMK / USD",
+          8,
+          "10129",
+          "0x72F88509C53b939a0613c679a0F4768c0444d247"
+        ],
+        "salt": "0xe124d7cc79a19705865fa21b784ba187cd393559e960c0c071132cb60354d1a3",
+        "feedId": "10129",
+        "base": null,
+        "quote": null
+      },
+      "10130": {
+        "address": "0x1EF2F2b7F6e91FAb0b2aEb726285dc7131fAA6D2",
+        "constructorArgs": [
+          "MXV / USD",
+          8,
+          "10130",
+          "0x72F88509C53b939a0613c679a0F4768c0444d247"
+        ],
+        "salt": "0xe124d7cc79a19705865fa21b784ba187cd393559e960c0c071132cb60354d1a3",
+        "feedId": "10130",
+        "base": null,
+        "quote": null
+      },
+      "10131": {
+        "address": "0x73B2421D0d1C001520AFF8e1AC5A873743EC1714",
+        "constructorArgs": [
+          "SLL / USD",
+          8,
+          "10131",
+          "0x72F88509C53b939a0613c679a0F4768c0444d247"
+        ],
+        "salt": "0xe124d7cc79a19705865fa21b784ba187cd393559e960c0c071132cb60354d1a3",
+        "feedId": "10131",
+        "base": null,
+        "quote": null
+      },
+      "10132": {
+        "address": "0xF0Fe671F4825A467B9ee604667bC100737fb6e78",
+        "constructorArgs": [
+          "XAF / USD",
+          8,
+          "10132",
+          "0x72F88509C53b939a0613c679a0F4768c0444d247"
+        ],
+        "salt": "0xe124d7cc79a19705865fa21b784ba187cd393559e960c0c071132cb60354d1a3",
+        "feedId": "10132",
+        "base": null,
+        "quote": null
+      },
+      "10133": {
+        "address": "0x43BE1920948Bf9C179A2365A0B752EA5eb733Ff4",
+        "constructorArgs": [
+          "XDR / USD",
+          8,
+          "10133",
+          "0x72F88509C53b939a0613c679a0F4768c0444d247"
+        ],
+        "salt": "0xe124d7cc79a19705865fa21b784ba187cd393559e960c0c071132cb60354d1a3",
+        "feedId": "10133",
+        "base": null,
+        "quote": null
+      },
+      "10134": {
+        "address": "0xd3D307A8F9F3f5274FDE6AE2A30cbfdf1F4AD208",
+        "constructorArgs": [
+          "XOF / USD",
+          8,
+          "10134",
+          "0x72F88509C53b939a0613c679a0F4768c0444d247"
+        ],
+        "salt": "0xe124d7cc79a19705865fa21b784ba187cd393559e960c0c071132cb60354d1a3",
+        "feedId": "10134",
+        "base": null,
+        "quote": null
+      },
+      "10135": {
+        "address": "0x9161FBb70ED476ddC355aCc105eB9b4be1347Efb",
+        "constructorArgs": [
+          "ZAC / USD",
+          8,
+          "10135",
+          "0x72F88509C53b939a0613c679a0F4768c0444d247"
+        ],
+        "salt": "0xe124d7cc79a19705865fa21b784ba187cd393559e960c0c071132cb60354d1a3",
+        "feedId": "10135",
+        "base": null,
+        "quote": null
+      },
+      "20000": {
+        "address": "0xedfa3ccA6C4A223f43c8781E29ad30f65C6A2D5D",
```

Comment:
The address here is different. Is that expected?

Reply:
Yes, on Pharos we are using our fork of Safe, because there isn't an official - so all contracts are with different addresses.